woocommerce/woocommerce
# Add product settings menu to new product management experience

### All Submissions:






### Changes proposed in this Pull Request:

Adds the product settings menu to the header of the new product management experience.

<img width="315" alt="Screen Shot 2022-11-05 at 11 56 48 AM" src="https://user-images.githubusercontent.com/10561050/201942852-db845283-4689-4352-acdf-16de6605673b.png">


Closes #35175  .






### How to test the changes in this Pull Request:



1. Navigate to the new product management experience ( Products -> Add new (MVP) )
2. Click on the Settings cog icon in the top right inside the header.
3. Note the menu opens with the option to enable reviews and update menu order
4. Toggle the “Advanced” section to make sure that it opens and closes as expected
5. Attempt to close the settings menu with both the cross (“x”) and the settings cog button.
6. Check that “Enable reviews” is checked by default on new products
7. Update the “Enable reviews” and “Menu order” settings and update or publish the product
8. Refresh the page and make sure that your settings persist.



### Other information:




### FOR PR REVIEWER ONLY:
